### DISCUSSION TITLE
Lock screen when screensaver is active?

### DISCUSSION THREAD

**Discussion by RonLinu** [SpiralLinux/SpiralLinux-project#73] [General]
I use Mate desktop on an older computer. Nice desktop, similar to Cinnamon, less ressource hungry but less configurable.
A small problem with its screensaver though: it does go into screensaving mode after the selected number of minutes but does not lock the screen even if that option is selected.

**Comment by tcitr-chen** [comment]
try to report in https://github.com/mate-desktop/mate-desktop/issues

**Reply by RonLinu** [reply]
Yes, you're right. It's a Mate issue.

**Comment by geckolinux** [comment]
> I use Mate desktop on an older computer. Nice desktop, similar to Cinnamon, less ressource hungry but less configurable.
> A small problem with its screensaver though: it does go into screensaving mode after the selected number of minutes but does not lock the screen even if that option is selected.

Hi there. Actually it's not a bug, it's configured to lock after 60 minutes, it's just a bit of a hidden setting in Mate:

![Screenshot from 2022-08-04 12-15-30](https://user-images.githubusercontent.com/2692138/182911335-e450d171-adff-4063-a94f-5e41d703f582.png)

**Reply by RonLinu** [reply]
Oh! I see. Thanks!
